doc/api/debugger.md

```diff
@@ -41,22 +42,23 @@ Once the debugger is run, a breakpoint will occur at line 3:
 
 ```txt
 $ node debug myscript.js
-< Debugger listening on 127.0.0.1:5858
-connecting to 127.0.0.1:5858 ... ok
-break in /home/indutny/Code/git/indutny/myscript.js:1
-> 1 global.x = 5;
+< Debugger listening on ws://127.0.0.1:9229/80e7a814-7cd3-49fb-921a-2e02228cd5ba
+< For help see https://nodejs.org/en/docs/inspector
+< Debugger attached.
```

Comment:
Weird, I'm seeing it on master/this branch:

<img width="695" alt="screen shot 2017-06-23 at 8 20 48 am" src="https://user-images.githubusercontent.com/567540/27488753-057089b4-57ed-11e7-9003-a2528e90df4d.png">

I also see it on 8.1.2:

<img width="680" alt="screen shot 2017-06-23 at 8 21 31 am" src="https://user-images.githubusercontent.com/567540/27488767-12f69b00-57ed-11e7-9910-4ec25a7ca848.png">

I'm taking a TODO to check where the message is generated and why it wouldn't happen consistently. Might be a timing and/or platform issue..?